.travis.yml

```diff
@@ -1,8 +1,7 @@
 rvm:
-  - 2.2.9
```

Comment:
I would suggest keeping this, as cucumber-rails still is using Ruby 2.2. If Rails 5.2 requires higher, then I suggest excluding it below in `.travis.yml`.